free-3.md

```diff
@@ -45,6 +54,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The [ASPC Scheduler](https://aspc.pomona.edu/courses/schedule/) kind of does this with class schedules in that it takes a schedule and formats it into a .csv or .ics to allow for easy importing. However, it is still a lot of clicking and querying for classes. If the user already knows what classes he or she is taking, and when they are, they should not have to go through the process of individually searching for the class and adding the class. 
 
```

Comment:
Similarly, someone built a [5C Scheduling App](https://github.com/bitsofpancake/hmc-scheduler) that still uses the browser but allows the user to add their own events to the class scheduling calendar.